source/fuzz/fuzzer_pass_merge_blocks.cpp

```diff
@@ -56,11 +56,7 @@ void FuzzerPassMergeBlocks::Apply() {
     uint32_t index = GetFuzzerContext()->RandomIndex(potential_transformations);
     auto transformation = potential_transformations.at(index);
     potential_transformations.erase(potential_transformations.begin() + index);
```

Comment:
@stefanomil separate from this PR, could you see whether "RemoveAtRandomIndex" from "FuzzerContext" could be used to make this piece of code simpler, and do a little PR for it if so?

Reply:
I will take a look